### ISSUE TITLE
[Bug] All-in players should not be marked as having folded

### ISSUE THREAD

**Issue by JbCourtois** [pokerwars/pokerwars.io-starterbot-python#6]
For what I understand from the rules of poker:

- **"All-in"** means a player has bet all their chips in the current hand ; they will not act in the betting phase but they can still win a side pot during the showdown
- **"Folded"** means a player has made the fold action: they are out of the hand and cannot win anything during the showdown

In Texas Hold'em, I think it is impossible for a player to be simultaneously all-in and folded, yet Pokerwars marks all-in players as folded.

**Comment by Under1ow** [comment]
Just yesterday I was at the casino with my husband, I want to say it was my first time and I did not really like it. It was too noisy, my husband is not allowed to smoke, I have to wear a mask, and I'm pregnant, and I'm nervous. My husband and I decided to sit at home today and play online casino [crazytime-casino.com](https://crazytime-casino.com/) . I so enjoyed lying at home in a cozy couch and watching my husband raise the family budget. Ohhh this is very cool. We lost $200 during the evening, but we got lucky, and the next spin gave us $2,000. This is the first time we won at the casino. Honestly, I loved it.

**Comment by runducoff** [comment]
You are correct in your understanding of the terms "all-in" and "folded" in poker. "All-in" means a player has bet all of their chips in the current hand, and they can still win the main pot and any side pots during the showdown. "Folded" means a player has decided to discard their hand and is out of the current hand, and they cannot win anything during the showdown. I learned it thanks to the courses at https://goit.global/ph/blog/training-company-goit-targets-filipinos-looking-to-change-careers-to-it/

**Comment by JohnAguiar1** [comment]
I love playing poker, and I have been doing it for a long time, so I have significant experience and play really well. However, after intense poker games, I always switch to the casino on [read](https://1wins.com.ph/) to relax. This site offers a vast selection of gambling games, including slots, roulette, and various table games. 1Win stands out for its convenience, reliability, and generous bonuses that make the gaming experience even more enjoyable. Thanks to its mobile app, I can play my favorite games anytime and anywhere, making my gaming experience even more pleasant.

**Comment by emiliacuppper** [comment]
nice

**Comment by Sirocco005** [comment]
Hey there! When it comes to poker rules, one essential detail that often sparks confusion is how to handle all-in players. On platforms like [888starz-bd.net](http://888starz-bd.net) , which offers a top-notch poker experience, the rules ensure clarity and fairness by treating all-in players correctly. All-in players should never be marked as having folded—once a player goes all-in, they're fully committed to the pot and remain active in the hand until the showdown. Their cards stay alive, and they're eligible to win the portion of the pot they can cover with their all-in amount. If you're passionate about poker or online gaming, choosing platforms that prioritize clear rules and fairness can make all the difference in your experience!
